baby_bingo.py

```diff
@@ -0,0 +1,60 @@
+

```

Comment:
I'd personally add a #! to the top of the file and maybe a comment about what version of python (i.e. 2.7 or 3)